tests/framework/ensembleModelTests/checkConnection/alpha.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import copy
+import time
+# CAUTION HERE #
+# IT IS IMPORTANT THAT THE USER IS AWARE THAT THE EXTERNAL MODEL (RUN METHOD)
+# NEEDS TO BE THREAD-SAFE!!!!!
+# IN CASE IT IS NOT, THE USER NEEDS TO SET A LOCK SYSTEM!!!!!
+import threading
+localLock = threading.RLock()
```

Comment:
you do not need the lock here (for this model). Please remove it and remove the comment above. In addition all the above imports are not used.